core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -119,6 +121,8 @@ public void start(WorkerNetAddress address) throws IOException {
             ImmutableMap.of(),
             ImmutableMap.of(),
             Configuration.getConfiguration(Scope.WORKER));
+        String dbDir = Configuration.getString(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_DIR);
+        mMetaStore = new RocksDBDoraMetaStore(dbDir + ".worker." + mWorkerId);
```

Comment:
close the metastore when the worker is closed.

Reply:
Oh, yes!

Reply:
fixed.